update/interface.go

```diff
@@ -0,0 +1,87 @@
+package update
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/epochStart/notifier"
+)
+
+type StateSyncer interface {
```

Comment:
why is this here?

Reply:
defines underlying methods.

Reply:
we should define interfaces in the (sub)packages where are used. Can remain like this, might be removed at a later time